pkg/cli/cmd/recipe/register/register.go

```diff
@@ -88,6 +89,7 @@ type Runner struct {
 	Workspace         *workspaces.Workspace
 	TemplateKind      string
 	TemplatePath      string
+	PlainHttp         bool
```

Comment:
Same comments apply here about the naming of the field. Initialisms like `HTTP` should be `ALLCAPS`.

Reply:
done